controllers/apply/building-connections/fields.js

```diff
@@ -45,49 +46,69 @@ const currentWork = [
 
 const yourIdea = [
     {
-        legend: 'Your project idea',
+        legend: 'Your idea',
         fields: [
             {
                 name: 'project-idea',
                 type: 'textarea',
-                lengthHint: LENGTH_HINTS.TOLSTOY,
+                lengthHint: LENGTH_HINTS.FEW_PARAS,
                 isRequired: true,
-                label: 'What is the project you would like funding for?',
-                helpText: `<ul>
-                    <li>Describe the project you would like funding for</li>
-                    <li>Tell us why it is needed in your area and how you will encourage collaboration [QUERY: What kind of collaboration?]</li>
-                </ul>`
+                label: 'What is the project you would like funding for, and why it is needed in your area?'
+            },
+            {
+                name: 'project-impact',
+                type: 'textarea',
+                lengthHint: LENGTH_HINTS.FEW_PARAS,
+                isRequired: true,
+                label: 'How will you scale up work you are already doing and/or join up in collaboration with others?'
+            },
+            {
+                name: 'project-activities',
+                type: 'textarea',
+                lengthHint: LENGTH_HINTS.FEW_PARAS,
+                isRequired: true,
+                label: 'Provide a brief outline of your delivery plan'
             }
         ]
     }
 ];
 
-const projectActivities = [
+const socialConnections = [
     {
-        legend: 'Project activities, outcomes, and milestones',
+        legend: 'Building social connections',
         fields: [
             {
-                name: 'project-activities-outline',
-                type: 'textarea',
-                lengthHint: LENGTH_HINTS.FEW_LINES,
-                isRequired: true,
-                label: 'Briefly outline what your project is aiming to achieve'
-            },
-            {
-                name: 'project-activities-first-quarter',
+                name: 'social-connections',
                 type: 'textarea',
-                lengthHint: LENGTH_HINTS.FEW_LINES,
+                lengthHint: LENGTH_HINTS.FEW_PARAS,
                 isRequired: true,
-                label: 'What will be achieved in the first quarter Jan—March 2019?'
-            },
+                label: 'How will the project help to build social connections and relationships between individuals and communities?',
+                helpText: `<ul>
+                    <li>Describe how the project will increase social connections, sense of community and/or belonging</li>
+                    <li>Describe how are people and communities involved in the project</li>
```

Comment:
I'd phrase this like:
"Describe how people and communities are involved in the project"

Reply:
Good shout.